dev/seq2seq/run_seq2seq_flax.py

```diff
@@ -866,7 +867,7 @@ def run_save_model(state, step, epoch, eval_metrics=None):
                 # log metrics
                 wandb_log(unreplicate(train_metric), step=global_step, prefix='train')
 
-            if global_step % training_args.eval_steps == 0:
+            if training_args.eval_steps and global_step % training_args.eval_steps == 0:
```

Comment:
it might not be necessary, `eval_steps` is set to 500 by default.

Reply:
I had an issue with it. Looks like it's None: https://huggingface.co/transformers/main_classes/trainer.html#trainingarguments

Reply:
yes, it's `None`, when it's `None` it's set to `logging_steps` which is 500 by default. You could find this info in the docstring.

Reply:
Apparently it works only if you set `evaluation_strategy` to `steps`, otherwise it stays to `None`